laravel/routes/web.php

```diff
@@ -41,69 +40,56 @@
     return Redirect::route('dashboard');
 });
 
-Route::get('/setup/installer/requirements', [RequirementsController::class, 'show_view'])->name('setup.installer.requirements');
-
-Route::get('/setup/installer/user', [UserController::class, 'show_view'])->name('setup.installer.user');
-Route::post('/setup/installer/user/add', [UserController::class, 'create'])->name('setup.installer.user.create');
+Route::middleware(['guest'])->group(function () {
+    Route::get('/setup/installer/requirements', [RequirementsController::class, 'show_view'])->name('setup.installer.requirements');
+    Route::get('/setup/installer/user', [UserController::class, 'show_view'])->name('setup.installer.user');
+    Route::post('/setup/installer/user/add', [UserController::class, 'create'])->name('setup.installer.user.create');
+});
 
 Route::middleware(['auth', 'verified'])->group(function () {
     Route::get('/dashboard', [DashboardController::class, 'dashboard'])->name('dashboard');
-
     Route::get('/instances', [InstanceController::class, 'overview'])->name('instances')->middleware('permission:view instances');
-    Route::get('/instance/add', [InstanceController::class, 'add'])->name('instance.add')->middleware('permission:add instances');
     Route::post('/instance/save', [InstanceController::class, 'save'])->name('instance.save')->middleware('permission:add instances');
-    Route::get('/instance/{instance_id}/edit', [InstanceController::class, 'edit'])->name('instance.edit')->middleware('permission:edit instances');
     Route::patch('/instance/{instance_id}/update', [InstanceController::class, 'update'])->name('instance.update')->middleware('permission:edit instances');
     Route::delete('/instance/{instance_id}/delete', [InstanceController::class, 'delete'])->name('instance.delete')->middleware('permission:delete instances');
-    Route::post('/instance/{instance_id}/start', [InstanceController::class, 'start'])->name('instance.start')->middleware('permission:start instances');
-    Route::post('/instance/{instance_id}/stop', [InstanceController::class, 'stop'])->name('instance.stop')->middleware('permission:stop instances');
-    Route::post('/instance/{instance_id}/restart', [InstanceController::class, 'restart'])->name('instance.restart')->middleware('permission:restart instances');
+    Route::get('/instance/{instance_id}/start', [InstanceController::class, 'start'])->name('instance.start')->middleware('permission:start instances');
```

Comment:
[CONCERN]

Mhmm, this still feels and is even technically wrong.

We're protecting the route additional with middlewares and permission checks, so these routes can't be missused.

However, there are technical reasons for using a specific HTTP method. [RFC 7231: 4.3 Method Definitions](https://datatracker.ietf.org/doc/html/rfc7231#section-4.3) defines and explains this very good.

We should stick to the RFC conform standards.

This applies to all routes, not only this single one here.

Reply:
Check